package.json

```diff
@@ -28,6 +28,11 @@
     "react-native-windows": "^0.55.0-rc.0"
   },
   "devDependencies": {
+    "eslint": "^3.19.0",
```

Comment:
Why are you adding a devDependency on eslint? We don't use this in any of our scripts. Plus, eslint doesn't provide much (any?) value in a project written in typescript. If you want to use eslint, feel free to install it on your system globally, but I don't want to add unnecessary dependencies to ReactXP.